helm/superset/templates/_helpers.tpl

```diff
@@ -63,8 +63,8 @@ CACHE_CONFIG = {
       'CACHE_KEY_PREFIX': 'superset_',
       'CACHE_REDIS_HOST': env('REDIS_HOST'),
       'CACHE_REDIS_PORT': env('REDIS_PORT'),
-      'CACHE_REDIS_DB': 1,
-      'CACHE_REDIS_URL': f"redis://{env('REDIS_HOST')}:{env('REDIS_PORT')}/1"
```

Comment:
isn't `CACHE_REDIS_URL` needed?

Reply:
Not if the other options are preset (it's essentially duplicating the above). Tested this config locally, and works without `CACHE_REDIS_URL`.